contrib/sgx/private_key_providers/source/BUILD

```diff
@@ -0,0 +1,62 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_contrib_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_contrib_package()
+
+envoy_cc_library(
+    name = "sgx_private_key_provider_lib",
+    srcs = [
+        "sgx.cc",
+        "sgx_private_key_provider.cc",
+        "utility.cc",
+    ],
+    hdrs = [
+        "pkcs11.h",
```

Comment:
Is it a common library? If so, please download it rather than put it there.

Reply:
@daixiang0 It's not a common library, it's the normative computer language definition file for PKCS11 v2.40 copied from [PKCS #11 v2.40 standard](http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/pkcs11-base-v2.40.html) .

Reply:
You can learn from below example:
```
def _com_github_google_perfetto():
    external_http_archive(
        name = "com_github_google_perfetto",
        build_file_content = """
package(default_visibility = ["//visibility:public"])
cc_library(
    name = "perfetto",
    srcs = ["perfetto.cc"],
    hdrs = ["perfetto.h"],
)
""",
    )
```

Reply:
Fixed by using external_http_archive to download pkcs11 header files